generative_models/cGAN_PyTorch/README.md

```diff
@@ -1,5 +1,7 @@
 # Pytorch Implementation of Conditional GAN 
+
```

Comment:
Remove cgan file from this pr

Reply:
Can you delete this pull request i made another one as I was not able to delete it